File: templates/Uwp/_composition/_shared/Page.ImageGalleryVB/Param_ProjectName/Services/ActivationService$wts.ItemName_gpostaction.vb
'{**
' This code block adds the call to `SampleDataService Initialize` in ActivationService of your project.
'**}
'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Namespace Services
    Friend Class ActivationService
        Private Async Function InitializeAsync() As Task
'{[{
            SampleDataService.Initialize("ms-appx:///Assets")
'}]}
        End Function
    End Class
End Namespace

File: templates/Uwp/_comp/_shared/Page.ImageGallery/Param_ProjectName/Services/ActivationService$wts.ItemName_gpostaction.cs
//{**
// This code block adds the call to `SampleDataService Initialize` in ActivationService of your project.
//**}
//{[{
using Param_RootNamespace.Core.Services;
//}]}
namespace Param_RootNamespace.Services
{
    internal class ActivationService
    {
        private async Task InitializeAsync()
        {
//{[{
            SampleDataService.Initialize("ms-appx:///Assets");
//}]}
        }
    }
}